blockchain/standalone/treasury.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package standalone
+
+import "fmt"
+
+// CalculateTSpendExpiry returns the only valid value relative to the next
+// block height where the transaction will expire. We add two blocks in the end
+// because transaction expiry is inclusive (>=) relative to blockheight.
+// We try very hard to use the "natural" types but this is a giant mess.
+func CalculateTSpendExpiry(nextBlockHeight int64, tvi, multiplier uint64) uint32 {
+	nbh := uint64(nextBlockHeight)
+	nextTVI := nbh + ((tvi - (nbh % tvi)) % tvi) // Round up to next TVI
+	if nextTVI == nbh {
+		nextTVI += tvi // NextBlockHeight cannot be itself.
+	}
+	maxTVI := nextTVI + tvi*multiplier // Max TVI allowed at this time.
+	return uint32(maxTVI + 2)
+}
+
+// IsTreasuryVoteInterval returns true if the passed height is on a Treasury
+// Vote Interval.
+func IsTreasuryVoteInterval(height, tvi uint64) bool {
+	return height%tvi == 0
```

Comment:
Im guessing this should be updated for `if height == 0 { return false } `

Reply:
Can't happen because genesis but I added the check.